src/components/Filter/Filter.tsx

```diff
@@ -13,10 +13,15 @@ import { FilterItems } from './filter-util';
 export const Filter: FC<FilterInterface> = ({
   availableFilters,
   filterData,
-  // onChange,
+  onChange,
 }) => {
   const [filterList, setFilterList] = useState<FilterListInterface[]>([]);
 
+  useEffect(() => {
+    if(filterList.length > 0)
+      onChange(FilterItems(filterData, filterList[0]));
```

Comment:
while what you have is totally fine - i think this is the first instance in our code where i've noticed an if statement without the `{` and `}` so just a minor tweak to add them for consistency with the rest of the code
```suggestion
    if(filterList.length > 0) {
      onChange(FilterItems(filterData, filterList[0]));
    }
```